main.go

```diff
@@ -14,50 +14,21 @@ import (
 	"github.com/antonlindstrom/mesos_stats"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/log"
+	"github.com/bolcom/mesos_metrics"
 )
 
 const concurrentFetch = 100
 
 // Commandline flags.
 var (
-	addr           = flag.String("web.listen-address", ":9105", "Address to listen on for web interface and telemetry")
-	autoDiscover   = flag.Bool("exporter.discovery", false, "Discover all Mesos slaves")
-	localURL       = flag.String("exporter.local-url", "http://127.0.0.1:5051", "URL to the local Mesos slave")
-	masterURL      = flag.String("exporter.discovery.master-url", "http://mesos-master.example.com:5050", "Mesos master URL")
-	metricsPath    = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics")
+	addr = flag.String("web.listen-address", ":9105", "Address to listen on for web interface and telemetry")
```

Comment:
ah, good one. Looks like the IntelliJ Golang plugin (https://github.com/go-lang-plugin-org/go-lang-idea-plugin) doesn't use `go fmt` yet. I shall start using `go fmt` :smile: